Sources/WebSockets/Serialization/FrameDeserializer.swift

```diff
@@ -3,13 +3,17 @@ import Transport
 
 public enum FrameParserError: Swift.Error {
     case missingByte
+    case packetTooLarge
```

Comment:
adding a case to a public enum is a breaking change. we'll need to use a new error here unfortunately